### DISCUSSION TITLE
[Feat]: Determine Genshin module content

### DISCUSSION THREAD

**Discussion by Sanielia** [Gwardia-Czapli/GwardiaHub#24] [RFC] [answered]
### My suggestion

We should decide what will be in Genshin Impact players module and:
- update `README.md` with this information
- update landing side with this information

**Comment by toriishia** [comment, marked as answer]
Here's some ideas:

-wish simulator (and collection book of the characters you already have)
-future banners info (duration, which characters)
-events info (duration, next events, rewards)
-daily hoyolab check in remainder (pop up notification)
-special ui design (later when the normal ui will be finished, genshin inspired)
